Packs/TheHiveProject/Integrations/TheHiveProject/TheHiveProject.yml

```diff
@@ -0,0 +1,1433 @@
+category: Case Management
+commonfields:
+  id: TheHiveProject
+  version: -1
+configuration:
+- additionalinfo: Ensure to include the port number with the URL
+  defaultvalue: https://soar.monstersofhack.com:9000
+  display: Host (e.g. https://soar.monstersofhack.com:9000)
```

Comment:
are you ok with this defaultvalue?

Reply:
@bakatzir Please could we change it to something like: http://IP_OF_MACHINE:9000?

In their OVA image download they suggest connecting with: http://IP_OF_VM:9000